composer.json

```diff
@@ -10,13 +10,29 @@
         "drupal/focal_point": "^1.5",
         "drupal/inline_entity_form": "^1.0-rc1",
         "drupal/svg_image": "^1.10",
-        "drupal/video_embed_field": "^2.0",
+        "drupal/video_embed_field": "dev-8.x-2.x as 2.5.0",
         "drupal/entity_usage": "^2.0@beta"
     },
     "repositories": {
         "drupal": {
             "type": "composer",
-            "url": "https://packages.drupal.org/8"
+            "url": "https://packages.drupal.org/8",
+            "canonical": false
+        },
+        "drupal/video_embed_field": {
+            "type": "vcs",
+            "no-api": true,
+            "url": "https://git.drupalcode.org/issue/video_embed_field-3311063.git"
+        }
+    },
+    "extra": {
+        "patches": {
+            "drupal/embed": {
+                "CKEditor 5 compatibility - https://www.drupal.org/project/embed/issues/3309747": "https://git.drupalcode.org/project/embed/-/merge_requests/2/diffs.patch"
```

Comment:
This is a dangerous patch, as one commit to the PR will change the diff. If you can get a diff from a specific commit hash that would be safer.

Reply:
hey @krakerag , Thanks for the review.
for fixing the compatibility issue for ck5, it has several commits, so it seems impossible to get a patch for a specific commit and we want to include all of the changes. 
please advice. 
https://git.drupalcode.org/project/embed/-/merge_requests/2/commits

Reply:
If someone pushes up another commit to that PR #2, your patch will change.

I would download the patch, add it as a comment to the issue, so that the uploaded patch cannot change. That's how we do it with regular patchfiles that don't use a PR.

Reply:
yes, that is correct. 
will do.